content/webapp/pages/user-panel.js

```diff
@@ -86,29 +86,21 @@ const UserPanelPage = () => {
                   v={{ size: 'l', properties: ['margin-bottom'] }}
                 >
                   <p className="font-hnm font-size-3">
-                    Is wellcomecollection.org working for you?
-                  </p>
-                  <p>
-                    {`Are you able to find the information you are looking for?
-                    Help improve our website, in-venue technology, and other
-                    services by answering questions like these as part of our
-                    user panel.`}
+                    Is wellcomecollection.org working for you? Can you find the
+                    information you’re looking for? Help improve our website,
+                    in-venue technology and other services by answering
+                    questions like these as part of our user panel.
                   </p>
+
                   <h2>Who can take part?</h2>
                   <p>
-                    {`We're looking for people who visit our exhibitions, read our
-                    stories, attend our events and use our library and
-                    collections - anyone who's interested in helping improve
-                    Wellcome Collection's services for everyone.`}
+                    {`We’re looking for people who visit our exhibitions, read our stories, attend our events and use our Library and collections – anyone who’s interested in helping us improve our services.`}
```

Comment:
We're using `’` and `'` differently, which should it be?

Reply:
good spot – I _think_ we tend to favour smart quotes – I'll check and update accordingly.

Reply:
I've not yet found a page where there has been consistent use of either `’` or `'`. Which suggests that this is something that needs broader consideration. But I've updated these to be all `’`.

Reply:
Opened #5348 so we can deal with this